exe_3.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+
+
+def number_of_rows_and_columns(matrix:np)->tuple:

```

Comment:
Use more specific types for type hinting.
https://numpy.org/devdocs/reference/typing.html